src/arrayMethodSort.js

```diff
@@ -1,11 +1,58 @@
+/* eslint-disable indent */
 'use strict';
 
 /**
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `__proto__` to extend the Array prototype is not recommended as it can lead to unexpected behavior and is considered bad practice. Instead, use `Array.prototype.sort2` to define the new method.